components/calendar/calendar.js

```diff
@@ -794,6 +789,10 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 		}
 	}
 
+	_onMousedown(e) {
+		e.preventDefault();
```

Comment:
What's this doing?

Reply:
This was since I switched the button to a div, if the button was disabled it could still receive focus when clicked so this prevented it. I've figured out that I can change the div back to a button so I'm going to get rid of that portion of the change.

Reply:
Cool sounds good!